### DISCUSSION TITLE
Renew a domain

### DISCUSSION THREAD

**Discussion by PedroTavolaro** [vercel/community#6604] [Help] [answered]
### Summary

My domain is blocked, i need a renew can you a help me ? domain: pedrotavolaro.com

### Example

pedrotavolaro.com

### Steps to Reproduce

My domain is blocked, i need a renew can you a help me ? domain: pedrotavolaro.com

**Comment by amyegan** [comment, marked as answer]
Hey @PedroTavolaro. Please [submit a **Domain Renewals** case](https://vercel.com/help) to the Customer Success Team. That will open a new ticket with the team that can help with this.

**Reply by PedroTavolaro** [reply]
ok, thank you very much @amyegan, I contacted support and opened a ticket, we are in the process of resolving it. Thank you : )

**Reply by ismailelbadawy** [reply]
I opened a similar case earlier today case number: #00254319

I need help responding to the case as i'm facing a lot of issues now and losing a client due to the bad process

**Reply by Cole-Krudwig** [reply]
I am also having this same issue and am worried I will not be able to renew my domain. I submitted a ticket #257274 but worried it isn't going through as I haven't received any sort of confirmation.

**Reply by amyegan** [reply]
Hey @Cole-Krudwig. Thanks for sharing the ticket number. I see that automatic domain renewal failed because of an problem with payment processing. The good news is the domain was still within the renewal grace period and I was able to manually renew for you. 

Please let me know if you have any questions!

[a comment by an automated account is not shown]
